Cargo.toml

```diff
@@ -233,78 +233,19 @@ resolver = "2"
 members = [
   "cmd/tikv-ctl",
   "cmd/tikv-server",
-  "components/api_version",
-  "components/backup",
-  "components/backup-stream",
-  "components/batch-system",
-  "components/causal_ts",
-  "components/cdc",
-  "components/cloud",
-  "components/cloud/aws",
-  "components/cloud/azure",
-  "components/cloud/gcp",
-  "components/codec",
-  "components/collections", "components/compact-log-backup",
-  "components/concurrency_manager",
-  "components/coprocessor_plugin_api",
-  "components/crossbeam-skiplist",
-  "components/crypto",
-  "components/encryption",
-  "components/encryption/export",
-  "components/engine_rocks_helper",
-  # Only enable tirocks in local development, otherwise it can slow down compilation.
-  # TODO: always enable tirocks and remove engine_rocks.
-  #  "components/engine_tirocks",
-  "components/error_code",
-  "components/external_storage",
-  "components/file_system",
-  "components/health_controller",
-  "components/into_other",
-  "components/keys",
-  "components/log_wrappers",
-  "components/online_config",
-  "components/panic_hook",
-  "components/pd_client",
-  "components/profiler",
-  "components/raftstore",
-  "components/raftstore-v2",
-  "components/resolved_ts",
-  "components/resource_control",
-  "components/resource_metering",
-  "components/security",
-  "components/server",
-  "components/service",
-  "components/snap_recovery",
-  "components/sst_importer",
-  "components/test_backup",
-  "components/test_coprocessor",
+  "components/*",
   "components/test_coprocessor_plugin/example_plugin",
-  "components/test_pd",
-  "components/test_pd_client",
-  "components/test_raftstore",
-  "components/test_raftstore-v2",
-  "components/test_raftstore_macro",
-  "components/test_sst_importer",
-  "components/test_storage",
-  "components/test_util",
-  "components/tidb_query_aggr",
-  "components/tidb_query_codegen",
-  "components/tidb_query_common",
-  "components/tidb_query_datatype",
-  "components/tidb_query_executors",
-  "components/tidb_query_expr",
-  "components/tikv_alloc",
-  "components/tikv_kv",
-  "components/tikv_util",
-  "components/tipb_helper",
-  "components/tracker",
-  "components/txn_types",
   "fuzz",
   "fuzz/fuzzer-afl",
   "fuzz/fuzzer-honggfuzz",
   "fuzz/fuzzer-libfuzzer",
   "tests",
 ]
+exclude = [
+  # `test_coprocessor_plugin` itself isn't a valid crate, so we exclude it to avoid match it by `components/*`.
+  # What is a valid crate is the subdir in `test_coprocessor_plugin`, like `example_plugin`.
+  "components/test_coprocessor_plugin",
```

Comment:
```suggestion
  "components/test_coprocessor_plugin",
  # Only enable tirocks in local development, otherwise it can slow down compilation.
  # TODO: always enable tirocks and remove engine_rocks.
  "components/engine_tirocks",
```